docs/release-notes.rst

```diff
@@ -74,6 +74,12 @@ Upcoming Version (Not Yet Released)
    in the "stable" docs (corresponding to the last release) but will be visible in the
    "latest" docs (corresponding to the master branch).
 
+Reloading in place
```

Comment:
The section names here are standardized (see "Types of Changes" above), so this would fall under "New Features". Also, each individual item is a sentence that describes the change; so this could be something like "`Flow` now has a `reload` method, which acts like `reloading` but modifies the flow in place."